### ISSUE TITLE
inconsistent handling of the alpha value in PointCloudDepthAndRGBtoXYZRGBA

### ISSUE THREAD

**Issue by csukuangfj** [PointCloudLibrary/pcl#2476]
## Your Environment

* PCL Version: the master branch

## Context

See the code
https://github.com/PointCloudLibrary/pcl/blob/9467b944ee3e30bb4cf8445d861c280f0fba1020/common/include/pcl/point_types_conversion.h#L373-L374

alpha is set to 0.
But
https://github.com/PointCloudLibrary/pcl/blob/9467b944ee3e30bb4cf8445d861c280f0fba1020/common/include/pcl/impl/point_types.hpp#L346-L350
https://github.com/PointCloudLibrary/pcl/blob/9467b944ee3e30bb4cf8445d861c280f0fba1020/common/include/pcl/impl/point_types.hpp#L352-L358

https://github.com/PointCloudLibrary/pcl/blob/9467b944ee3e30bb4cf8445d861c280f0fba1020/common/include/pcl/impl/point_types.hpp#L560-L566

https://github.com/PointCloudLibrary/pcl/blob/9467b944ee3e30bb4cf8445d861c280f0fba1020/common/include/pcl/impl/point_types.hpp#L627-L633

https://github.com/PointCloudLibrary/pcl/blob/9467b944ee3e30bb4cf8445d861c280f0fba1020/common/include/pcl/impl/point_types.hpp#L634-L642

https://github.com/PointCloudLibrary/pcl/blob/9467b944ee3e30bb4cf8445d861c280f0fba1020/common/include/pcl/impl/point_types.hpp#L659-L666

https://github.com/PointCloudLibrary/pcl/blob/9467b944ee3e30bb4cf8445d861c280f0fba1020/common/include/pcl/impl/point_types.hpp#L667-L676

all set alpha to 255.
## Expected Behavior

For consistency, alpha in PointCloudDepthAndRGBtoXYZRGBA should be set to 255 instead of 0.
## Current Behavior

alpha in PointCloudDepthAndRGBtoXYZRGBA is set to 0.
## Code to Reproduce

See the code above.
## Possible Solution

Set alpha in PointCloudDepthAndRGBtoXYZRGBA to 255.

**Comment by taketwo** [comment]
Agree, alpha should be 255. Since it is already initialized inside the constructor, line 374 may be removed altogether.
